packages/mdc-textfield/README.md

```diff
@@ -149,210 +135,100 @@ See [here](icon/) for more information on using icons.
 </div>
 ```
 
-### Full-width
+#### Disabled
```

Comment:
I think we can remove this, it seems that a line under "CSS Classes" will suffice for any variations that don't require further explanation.

Reply:
I added some more notes about why this is in the "HTML Structure" section. Basically you need a 'disabled' attribute.